nanoFramework.Json/JsonObject.cs

```diff
@@ -16,79 +16,11 @@ internal class JsonObject : JsonToken
     {
         private readonly Hashtable _members = new();
 
-        public JsonProperty this[string name]
-        {
-            get { return (JsonProperty)_members[name.ToLower()]; }
-            set
-            {
-                if (name.ToLower() != value.Name.ToLower())
-                {
-                    throw new ArgumentException("index value must match property name");
-                }
-
-                _members.Add(value.Name.ToLower(), value);
-            }
-        }
-
-        public bool Contains(string name) => this._members.Contains(name.ToLower());
```

Comment:
Here's another one that seems useful when navigating through members.

Reply:
It's inside internal class without any references, so it's not used in JSON project. 
In my opinion - if it's internal and not used it should be deleted (to minimalize deployment size of dll).

Reply:
OK. Let it go then!